tests/container/vnc-script.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# vnc-script
```

Comment:
Could we name this something a bit more obvious? vnc-script.sh has me thinking it isn't just for a test.